docs/monorepo.md

```diff
@@ -158,101 +161,39 @@ Note that if you're building with Webpack, you may need to turn off [`resolve.sy
 
 ## Publishing
 
-We use [Lerna](https://lerna.js.org/) and its `publish` command to publish the monorepo packages to NPM registry. Please do not use regular [`yarn publish`](https://docs.npmjs.com/cli/publish) within a package to publish an individual package; `npx` has issues using this flow.
+We use the regular NPM publish flow to publish packages.
```

Comment:
Can this section be removed? It doesn't really say anything interesting. It's the "Publishing a package" below that has the same topic and contains useful information.